Carbon.IIS/Functions/Install-CIisAppPool.ps1

```diff
@@ -36,48 +36,150 @@ function Install-CIisAppPool
     application pool is created that is 32-bit, uses .NET 2.0, and a classic pipeline.
     #>
     [OutputType([Microsoft.Web.Administration.ApplicationPool])]
+    [CmdletBinding(DefaultParameterSetName='New')]

```

Comment:
New becomes not-New. Is more applicable refinement?

Reply:
This is just for the v1 major release and is a way of maintaining backwards compatibility. I want code that's using the old parameters to continue to work (the `Deprecated` parameter set), but don't want anyone using both deprecated and new parameters in the same call. In v2, all the deprecated parameters will be removed, and I'll remove the `New` parameter set because there will only be one parameter set.